frontend/src/routes/manage/Video/VideoDetails.tsx

```diff
@@ -98,3 +105,12 @@ const VideoButtonSection: React.FC<{ event: AuthorizedEvent }> = ({ event }) =>
     </ButtonSection>;
 };
 
+const VideoMetadataSection: React.FC<{ event: AuthorizedEvent }> = ({ event }) => {
+    const [commit, inFlight] = useMutation<VideoDetailsMetadataMutation>(updateEventMetadata);
+
+    return <MetadataSection
+        item={{ ...event, description: event.description }}
```

Comment:
Why not simply `item={event}`?

Reply:
Left over from the mismatching placement of description fields in series and event API definitions, I believe.